src/dplasmaaux_cuda.h

```diff
@@ -23,40 +23,26 @@
 #include <cublas_v2.h>
 #endif  /* !defined(CUBLAS_V2_H_) */
 
```

Comment:
These don't need to be expressions returning a value (which I think it's a GNU extension), they can be statements macros surrounded by `do {} while (0)`.